PharoObjectModel/PharoObjectModel.pier

```diff
@@ -292,56 +296,66 @@ Dog class>>>count
     ^ count
 ]]]
 
```

Comment:
We are missing the class class definition with the count class instance variable names.